scripts/reindex.py

```diff
@@ -58,6 +62,13 @@
                     ],
                     choices=config.catalogs,
                     help='The names of the catalogs to reindex.')
+parser.add_argument('--sources',
+                    nargs='*',
+                    default=(),
```

Comment:
```suggestion
                    default=('*'),
```

And then we'll use globbing to match. This will make it easier to specify multiple source that share a name prefix. Special care must be taken to avoid these globs from being consumed by the shell.

Reply:
It'll help to make this script aware of `azul_reindex_sources`, so we don't need to modify the makefile. So something along the lines of

```suggestion
                    default=os.environ.get('azul_reindex_sources', '*').split(','),
```